sandpack-react/src/contexts/sandpackContext.tsx

```diff
@@ -294,6 +300,14 @@ class SandpackProvider extends React.PureComponent<
     // Clear the queued listeners after they were registered
     this.queuedListeners = {};
     this.setState({ sandpackStatus: "running" });
+
+    if (this.timeoutHook) {
```

Comment:
this is more like a fail safe, in case the run function is called multiple times for whatever reason